home/static/home/js/explore_page_js.js

```diff
@@ -0,0 +1,30 @@
+
+
```

Comment:
How do you test this JavaScript code?

Reply:
@ifireball

The JavaScript file only change the color of the selected item and I checked it manually, do I need to check it automatically as well ?